tests/test_flow/test_persistence_fuzz.py

```diff
@@ -181,25 +208,25 @@ def add_entities(self, n_entities):
             all_names = self.model.entity_names()
 
             dep_names = []
+            changes_per_run = self._random_bool()
```

Comment:
I like the idea of reducing the frequency of nondeterministic nodes instead of adding another test. FWIW, `test_small_fixed_flow_short_fuzz` does not use `changes_per_node`.